SignalServiceKit/src/Storage/OWSStorage.m

```diff
@@ -243,37 +248,107 @@ - (instancetype)initStorage
 {
     self = [super init];
 
-    if (![self tryToLoadDatabase]) {
-        // Failing to load the database is catastrophic.
-        //
-        // The best we can try to do is to discard the current database
-        // and behave like a clean install.
-        OWSProdCritical([OWSAnalyticsEvents storageErrorCouldNotLoadDatabase]);
+    if (self) {
+        if (![self tryToLoadDatabase]) {
+            // Failing to load the database is catastrophic.
+            //
+            // The best we can try to do is to discard the current database
+            // and behave like a clean install.
+            OWSProdCritical([OWSAnalyticsEvents storageErrorCouldNotLoadDatabase]);
 
-        // Try to reset app by deleting database.
-        // Disabled resetting storage until we have better data on why this happens.
-        // [self resetAllStorage];
+            // Try to reset app by deleting all databases.
+            [OWSStorage deleteDatabaseFiles];
```

Comment:
Just making sure this was intentionally enabled since it was previously commented out, and why the switch from `resetAllStorage` -> `deleteDatabaseFiles`

Seems like we'll have a lot of orphaned data, e.g. attachments, profiles

Reply:
Re-commented it with a TODO to revisit other cleanup.